src/averageAges.js

```diff
@@ -16,6 +16,13 @@
  */
 function calculateMenAverageAge(people, century) {
   // write code here
+  let men = people.filter(({ sex }) => sex === 'm');
+
+  century !== undefined && (
+    men = men.filter(({ died }) => Math.ceil(died / 100) === century)
+  );
```

Comment:
will be better to use ternary operator here, and assign the result of the correct filter call to the men variable